submission/pipelines/generate_pipelines.py

```diff
@@ -1,6 +1,7 @@
 import json
 import os
-
+import pymongo
```

Comment:
This doesn't come pre-installed.  Do we want a requirements.txt file?

Reply:
sure